src/server/frontend_wayland/xdg_output_v1.cpp

```diff
@@ -112,8 +112,11 @@ mf::XdgOutputV1::XdgOutputV1(
     // TODO: Better output names that are consistant between sessions
     auto output_name = "OUT-" + std::to_string(output_global.current_config().id.as_value());
     send_name_event_if_supported(output_name);
+    send_done_event();
```

Comment:
This is incorrect - `xdg_output_v1` is not the same interface as `wl_output`, and [as per the protocol](https://wayland.app/protocols/xdg-output-unstable-v1#zxdg_output_v1:event:done) the `done` event on `xdg_output_v1` is deprecated and should not be sent on ≥v3 objects.